### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by kevin-aus**
Please check formatting, naming and logic, \@cue-aus

### CODE REVIEW

File: lec04/activity2.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+countries = ['China', 'India', 'United States', 'Indonesia', 'Pakistan']
+populations = [1439323776, 1380004385, 331002651, 273523615, 220892340]
+no_of_countries = len(countries)
+first_letters = []
```
Comment (cue-aus) [inline lec04/activity2.py:4]: code is not completed

File: lec04/activity2_first_way.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+for i in range(0, no_of_countries):
+    print(str(i+1)+'.',countries[i])
+
+do_again = 'Y'
+while do_again == 'Y':
+    choice = int(input('Enter a number from 1 to 5: '))
+    print('Population of',countries[choice-1],'is',populations[choice-1])
+    do_again = input('Enter y to continue or anything else to quit: ')
+    do_again = do_again.upper()
+print('Bye.')
```
Comment (cue-aus) [inline lec04/activity2_first_way.py]: No new line at the end

File: practicals/emails.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+def get_name_from_email(email):
+    """Extract expected name from email address."""
+    prefix = email.split('@')[0]
+    parts = prefix.split('.')
+    name = " ".join(parts).title()
+    return name
+
+
+main()
```
Comment (cue-aus) [inline practicals/emails.py]: No new line at the end

File: lec04/activity2.py (hunk 1 above)
Comment (kevin-aus) [reply lec04/activity2.py:4]: it intends to do so

**Review by cue-aus — APPROVED** [review]

**Pull request merged**